test/features/forms/MainDonation_UpgradeToMonthlyOrYearly.ts

```diff
@@ -0,0 +1,92 @@
+import { VueWrapper } from '@vue/test-utils';
+import {
+	expectMainDonationFormGoesToPageOnSubmit,
+	expectMainDonationFormSubmits,
+	submitMainDonationForm
+} from '@test/features/forms/subForms/MainDonationForm';
+import {
+	expectUpgradeToYearlyFormGoesToPageOnLinkClick,
+	expectUpgradeToYearlyFormSubmits
+} from '@test/features/forms/subForms/UpgradeToYearlyForm';
+import {
+	expectUpgradeToMonthlyFormGoesToPageOnLinkClick,
+	expectUpgradeToMonthlyFormSubmits
+} from '@test/features/forms/subForms/UpgradeToMonthlyForm';
+import { Intervals } from '@src/utils/FormItemsBuilder/fields/Intervals';
+import { PaymentMethods } from '@src/utils/FormItemsBuilder/fields/PaymentMethods';
+
+enum Pages {
+	MainDonation = 1,
+	UpgradeToYearly = 2,
+	UpgradeToMonthly = 3
+}
+
+export const donationFormFeatures: Record<string, ( wrapper: VueWrapper<any> ) => Promise<any>> = {
+	expectMainDonationFormSubmitsWhenSofortIsSelected: ( wrapper: VueWrapper<any> ) => expectMainDonationFormSubmits(
+		wrapper,
+		Intervals.ONCE,
+		PaymentMethods.SOFORT,
+		'5'
+	),
+	expectMainDonationFormSubmitsWhenYearlyIsSelected: ( wrapper: VueWrapper<any> ) => expectMainDonationFormSubmits(
+		wrapper,
+		Intervals.YEARLY,
+		PaymentMethods.PAYPAL,
+		'5'
+	),
+	expectMainDonationFormSubmitsWhenMonthlyIsSelected: ( wrapper: VueWrapper<any> ) => expectMainDonationFormSubmits(
+		wrapper,
+		Intervals.MONTHLY,
+		PaymentMethods.PAYPAL,
+		'5'
+	),
+	expectMainDonationFormSubmitsWhenAmountIsGreaterThanMaxAmount: async ( wrapper: VueWrapper<any> ) => expectMainDonationFormSubmits(
+		wrapper,
+		Intervals.YEARLY,
+		PaymentMethods.PAYPAL,
+		'25'
+	),
+	expectMainDonationFormGoesToUpgradeYearlyOnHighAmounts: ( wrapper: VueWrapper<any> ) => expectMainDonationFormGoesToPageOnSubmit( wrapper,
+		Pages.UpgradeToYearly,
+		Intervals.ONCE,
+		PaymentMethods.PAYPAL,
+		'200'
```

Comment:
To fix this test you need to add a €200 option to the form items in `test/banners/formItems.ts`